hexa/pipelines/schema.py

```diff
@@ -270,13 +271,22 @@ def resolve_create_pipeline(_, info, **kwargs):
         }
 
     try:
+        if input.get("timeout") and input.get("timeout") > int(
+            os.environ.get("PIPELINE_RUN_MAX_TIMEOUT")
```

Comment:
You should use the Django settings instead of directly accessing the env variable (`from django.conf import settings`)